apps/design/frontend/src/api.ts

```diff
@@ -273,17 +264,75 @@ export const getBallotTemplate = {
   },
 } as const;
 
+/* istanbul ignore next - WIP @preserve */
+export const ttsEditsGet = {
+  queryKey(params: TtsEditKey): QueryKey {
+    return ['ttsEditsGet', params.orgId, params.languageCode, params.original];
+  },
+  useQuery(params: TtsEditKey, opts: { enabled?: boolean } = {}) {
+    const apiClient = useApiClient();
+    return useQuery(
+      this.queryKey(params),
+      () => (params.original ? apiClient.ttsEditsGet(params) : null),
```

Comment:
When would `params.original` not be truthy? Just curious - it's sometimes hard to understand these chunks without their usage context

Reply:
Also leftover from the demo code (where everything was just getting fetched at the top level and a key may or may not have been selected yet) - somehow missed this query while cleaning up

> it's sometimes hard to understand these chunks without their usage context

LMK if there's anything else I can clarify around this or the upcoming feature in general